docs/en/observability/observability-ui.asciidoc

```diff
@@ -71,54 +70,58 @@ possible memory leaks and whether disk resources need to be added or redistribut
 outbound traffic metrics allow you to visualize any possible spikes to help determine whether
 any hosts are choking your network.
 
-To drill-down and view these performance metrics in the *Metrics app*, click *View in app*.
+To drill-down and view these performance metrics in the *{metrics-app}*, click *Show inventory*.
 For more information, see <<analyze-metrics,Analyze metrics>>.
 
 //TODO: what are the specific metric fields?
 
 [role="screenshot"]
-image::images/metrics-summary.png[Metrics summary]
+image::images/metrics-summary.png[Summary of Hosts on the {observability} overview page]
+
+[float]
+[[view-services-and-transactions]]
+== Services
+
+The *Services* chart displays the total number of services running within your environment
+and the total number of transactions per minute that were captured by the Elastic APM
+agent instrumenting those services.
+
+To drill-down and view this APM data in the *{apm-app}*, click *Show service inventory*.
```

Comment:
```suggestion
To drill down and view this APM data in the *{apm-app}*, click *Show service inventory*.
```
I think I got the review nudge to change "drill-down" to "drill down" in a previous PR. Not highlighted all occurences here.

Reply:
This is something we could lint for in the future. 🙂